src/components/results/result.js

```diff
@@ -47,8 +48,8 @@ class FederatedResult extends React.Component {
     if (Object.hasOwnProperty.call(doc, 'sm_urls') && Array.isArray(doc.sm_urls)) {
       // Attempt to find a url from the current host.
       const currentHostUrl = doc.sm_urls.find((item) => {
-        const url = new URL(item);
```

Comment:
IE doesn't support this constructor, see MDN: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#Browser_compatibility

Unfortunately MDN doesn't provide a polyfill, my searches lead me to use the Url NPM package which is what we do in this PR